project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx

```diff
@@ -96,15 +105,24 @@ export const MenuIconicItemUserPopover: FC<MenuIconicItemUserPopoverProps> = ({
     isAuthenticated,
     children,
 }) => {
-    const positionClasses = isAuthenticated ? '-right-[100%]' : 'right-0';
+    const isDesktop = useMediaMin('vl');
+
+    if (!isDesktop) {
+        return null;
+    }
+
+    const positionClasses = isAuthenticated
+        ? '-right-[100%] min-w-[355px]'
+        : 'right-0 max-w-[335px] lg:right-[-180px] lg:min-w-[740px] vl:min-w-[807px]';
 
     return (
         <AnimatePresence initial={false}>
             {isHovered && (
                 <AnimateAppearDiv
                     className={twMergeCustom(
-                        `pointer-events-auto absolute ${positionClasses} top-[54px] z-cart hidden ${isAuthenticated ? 'min-w-[355px]' : 'max-w-[335px] lg:right-[-180px] lg:min-w-[740px] vl:min-w-[807px]'} origin-top`,
-                        'rounded-xl bg-background p-5 lg:block',
+                        `pointer-events-auto absolute ${positionClasses} top-[54px] z-cart hidden origin-top`,
```

Comment:
`positionClasses` should be added as second parameter to `twMergeCustom` in my opinion.

Reply:
fixed